src/app/contextBroker/contextBroker.cpp

```diff
@@ -1567,7 +1549,7 @@ int main(int argC, char* argV[])
   SemRequestType policy = policyGet(reqMutexPolicy);
   orionInit(orionExit, ORION_VERSION, policy, mutexTimeStat);
   mongoInit(dbHost, rplSet, dbName, user, pwd, dbTimeout, writeConcern, dbPoolSize, mutexTimeStat);
-  contextBrokerInit(ngsi9Only, dbName, mtenant);
+  contextBrokerInit(dbName, mtenant);
   curl_global_init(CURL_GLOBAL_NOTHING);
 
```

Comment:
Isn't there a vector of services also, in this file?
The one used when running as CM ...
Or, have we cleaned that up already?